doc/sql.extensions/README.context_variables2

```diff
@@ -131,6 +131,8 @@ Usage:
    EXT_CONN_POOL_ACTIVE_COUNT   | Count of active connections, associated with pool
                                 |
    EXT_CONN_POOL_LIFETIME       | Idle connection lifetime, in seconds
+                                |
+   SESSION_TIME_ZONE            | Current session time zone.
```

Comment:
Given we have TIMEZONE_* keywords, *TIMESTAMP and LIFETIME (all without space in the middle), I'd suggest to change the name to _SESSION_TIMEZONE_.